### DISCUSSION TITLE
Discovery in Wireguard network

### DISCUSSION THREAD

**Discussion by JonathanBouligny** [rustdesk/rustdesk#7726] [General]
Not sure if this is a network configuration issue but it appears the discovery of machines doesnt work when the machines are conencted via wireguard. This it only important because i wanted to use the wake on lan feature which is tied to the discovery feature. I can connect to the other machine and through the relay server which all machines make connections to via wireguard but it doesnt discover any machines on lan. Im not sure if i can point the discovery feature at a specific subnet because wireguard should be considered a lan network to discover on in this case.

Thank you!

**Comment by adamzman** [comment]
+1

**Comment by xlionjuan** [comment]
Not possible
